documentation/src/main/asciidoc/topics/ref_discovery_azure.adoc

```diff
@@ -1,5 +1,6 @@
+[id='discovery_azure-{context}']
 = AZURE_PING
```

Comment:
@tristantarrant we don't productize this and we don't ship it with the server. should be removed from server docs? (or include it in the server? or document how to include it in the server xD)